src/platform/silabs/tracing/BUILD.gn

```diff
@@ -63,3 +68,30 @@ static_library("SilabsTracing") {
     "${chip_root}/src/tracing:tracing",
   ]
 }
+
+static_library("SilabsPowerTracing") {
+  sources = [
+    "SilabsPowerTracing.cpp",
+    "SilabsPowerTracing.h",
+  ]
+
+  deps = [
+    "${chip_root}/src/lib/core",
+    "${chip_root}/src/lib/support",
+    "${chip_root}/src/system",
+  ]
+
+  public_deps = [
+    ":silabs_tracing_config",
+    "${chip_root}/src/tracing:tracing",
+  ]
+}
+
+static_library("SilabsPowerTracingStubs") {
```

Comment:
Maybe a small comment or rename that indicates that those mocks are for tests.
Also, a small question. Why are we choosing  `static_library`  here (and line 72) rather than `source_set`